payloads/library/exfiltration/iMessageGrabber/readme.md

```diff
@@ -0,0 +1,28 @@
+# iMessageGrabber
+---
+- Author: GermanNoob
+- Creds: wiredBrother, who did a RubberDucky payload for iMessage on MacOS
+
+### Description
+---
+This payload grabs the iMessage archive of the current user, packs it to a zip file  and moves it to the BashBunny /loot/iMessage/ folder
+
+### Dependencies
+---
+None :)
+
+### Configuration (optional)
+---
+If you have renamed your BashBunny storage device, you have to change the name in the options section
+
```

Comment:
In order to access Messages content, terminal has to have full disk access in MacOS-X. Please specify this inside of your read me.